compat-furnaces.lua

```diff
@@ -330,57 +55,13 @@ minetest.register_node(":default:furnace", {
 		input_inventory = "dst",
 		connect_sides = {left = 1, right = 1, back = 1, front = 1, bottom = 1, top = 1}
 	},
-	paramtype2 = "facedir",
-	legacy_facedir_simple = true,
-	is_ground_content = false,
-	sounds = default.node_sound_stone_defaults(),
-
-	can_dig = can_dig,
-
-	on_timer = furnace_node_timer,
 
-	on_construct = function(pos)
-		local meta = minetest.get_meta(pos)
-		meta:set_string("formspec", inactive_formspec(pos, meta))
-		local inv = meta:get_inventory()
-		inv:set_size('src', 1)
-		inv:set_size('fuel', 1)
-		inv:set_size('dst', 4)
-	end,
-
-	on_metadata_inventory_move = function(pos)
-		minetest.get_node_timer(pos):start(1.0)
-	end,
-	on_metadata_inventory_put = function(pos)
-		-- start timer function, it will sort out whether furnace can burn or not.
-		minetest.get_node_timer(pos):start(1.0)
-	end,
-	on_blast = function(pos)
-		local drops = {}
-		default.get_inventory_drops(pos, "src", drops)
-		default.get_inventory_drops(pos, "fuel", drops)
-		default.get_inventory_drops(pos, "dst", drops)
-		drops[#drops+1] = "default:furnace"
-		minetest.remove_node(pos)
-		return drops
-	end,
-	allow_metadata_inventory_put = allow_metadata_inventory_put,
-	allow_metadata_inventory_move = allow_metadata_inventory_move,
-	allow_metadata_inventory_take = allow_metadata_inventory_take,
-	on_receive_fields = function(pos, formname, fields, sender)
-		if not pipeworks.may_configure(pos, sender) then return end
-		fs_helpers.on_receive_fields(pos, fields)
-		local meta = minetest.get_meta(pos)
-		local formspec = inactive_formspec(pos, meta)
-		meta:set_string("formspec", formspec)
-	end,
 	after_place_node = pipeworks.after_place,
 	after_dig_node = pipeworks.after_dig,
 	on_rotate = pipeworks.on_rotate
 })
 
-minetest.register_node(":default:furnace_active", {
-	description = DS("Furnace"),
+minetest.override_item("default:furnace_active", {
 	tiles = {
 		"default_furnace_top.png"..tube_entry,
 		"default_furnace_bottom.png"..tube_entry,
```

Comment:
tiles could also be overridden instead of replaced, but that probably matters so little there's no reason to bother

Reply:
mainly didnt bother, because of the different numbers of textures you can have, if someone really wants, they can submit a PR for this. else its a non issue